github/authorizations.go

```diff
@@ -433,3 +433,30 @@ func (s *AuthorizationsService) DeleteImpersonation(ctx context.Context, usernam
 
 	return s.client.Do(ctx, req, nil)
 }
+
+// RevokedAppAuthorizationEvent checks if authorization for an application has been revoked.
+//
+// NOTE: if your GitHub App continues to use a revoked access token,
+// it will receive the 401 Bad Credentials error.
+//
+// GitHub API docs: https://developer.github.com/v3/activity/events/types/#githubappauthorizationevent
+func (s *AuthorizationsService) RevokedAppAuthorizationEvent(ctx context.Context, user string, opt *ListOptions) ([]*Event, *Response, error) {
```

Comment:
This looks like it's just returned the list of events for the given user which seems different to the function name and description.

I would expect this to be implemented in an events service.

https://developer.github.com/v3/activity/events/

Reply:
I initially had it there but @gmlewis reviewed and recommended I move it to `authorizations.go`. 

Another thing to note @gauntface , this is not available in the events API as stated in the [description](https://developer.github.com/v3/activity/events/types/#githubappauthorizationevent): `This event is not available in the Events API.`. Despite that fact, do you still think this should be implemented as an events service?

Reply:
I think this function is mixing the events API (https://developer.github.com/v3/activity/events/#list-events-performed-by-a-user) which we would normally have under an events service and the Revoked Auth WebHook event (https://developer.github.com/v3/activity/events/types/#githubappauthorizationevent) which I would expect the golang library to be able to parse the json into a struct.

I believe that is was @gmlewis has done in his PR. But not sure if that is what you are looking for.